.github/workflows/main.yaml

```diff
@@ -52,15 +52,11 @@ jobs:
     steps:
       - name: Vault Secrets
         id: secrets
-        uses: hashicorp/vault-action@7d98524254c38dc6892804f991526ff30905f643 # tag=v2.4.2
+        uses: SonarSource/vault-action-wrapper@299f5550eeec99332c9e8b55b0e77ec77fe9cf34  # tag=2.4.2
         with:
-          url: ${{ env.VAULT_ADDR }}
-          exportEnv: false
-          method: jwt
-          path: jwt-ghwf
-          role: ${{ format('github-{0}-{1}', github.repository_owner, github.event.repository.name) }}
+          url: ${{ inputs.vaultAddr }}
           secrets: |
-            ${{ format('development/artifactory/token/{0}-{1}-{2}', github.repository_owner, github.event.repository.name, env.ARTIFACTORY_ROLE_SUFFIX) }} access_token | artifactory_access_token;
+            development/artifactory/token/REPO_OWNER_NAME_DASH-${{ inputs.artifactoryRoleSuffix }} access_token | artifactory_access_token;
```

Comment:
replace plain text REPO_OWNER_NAME_DASH by something showing that it's going to be replaced, maybe add a comment also as where it's going to be replaced